daemon/container.go

```diff
@@ -1036,6 +1036,9 @@ func (container *Container) networkMounts() []execdriver.Mount {
 }
 
 func (container *Container) addLocalMountPoint(name, destination string, rw bool) {
+	if container.MountPoints == nil {
+		container.MountPoints = map[string]*mountPoint{}
```

Comment:
we actually initialize that mountPoints when a new container is created:

https://github.com/coolljt0725/docker/blob/fix_panic_in_verifyoldvolumesinfo/daemon/daemon.go#L543

I guess not all logic goes through that function.